.vscode/launch.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
I don't see any security issues, or leak of personal information, with the file, so I don't see any issues on that front. I personally am finding that since a majority of team members as using VS Code, it is very helpful to share this file for source code debugging. It was created manually. If you have a strong objection, I will remove it.

Reply:
No, as long as it's not developer-specific, it sounds like it'll be useful.